apps/super-app/src/common/components/ServiceCard.tsx

```diff
@@ -0,0 +1,57 @@
+import './ServiceCard.css';
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  Typography,
+  Box,
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import { ServiceCardAbout } from '../../services';
+
+export function ServiceCard(props: ServiceCardAbout) {
+  const { logo, name, shortBlurb, route } = props;
+  return (
+    <Card sx={{ display: 'flex', marginBottom: '16px' }}>
+      <CardMedia
+        component="img"
+        sx={{ width: '5rem', height: '5rem', margin: 'auto 16px' }}
+        image={logo}
+        alt="TWC2 logo"
```

Comment:
Shouldn't this come from the props too? At any rate, I'm also okay to omit the `alt`, I don't think it will matter in practice.

Reply:
HAHA GOOD CATCH